guides/writing_shards.md

```diff
@@ -14,9 +14,9 @@ In this tutorial, we'll be making a Crystal library called _palindrome-example_.
 ### Requirements
 
 In order to release a Crystal Shard, and follow along with this tutorial, you will need the following:
-* A working installation of the [Crystal compiler](../using_the_compiler/README.md)
-* A working installation of [Git](https://git-scm.com)
-* A [GitHub](https://github.com) account
+* A working installation of the [Crystal compiler](../using_the_compiler/README.md).
+* A working installation of [Git](https://git-scm.com).
+* A [GitHub](https://github.com) or [GitLab](https://gitlab.com/) account.
```

Comment:
Adding full stops to the list items here should be another PR unifying typographical style of lists.